earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp

```diff
@@ -649,6 +649,10 @@ void RasterAssetWidget::AssembleEditRequest(
   } else {
     request->config.provider_id_ = 0;
   }
+
+  if (!acquisition_date_wrapper_->IsValidDate()) {
```

Comment:
I also tried saving with the valid date of 2000-01-01 and it still gives me the error saying I have entered an invalid date.

Reply:
This was tested on CentOS 7.5